src/ansys/hps/client/jms/api/project_api.py

```diff
@@ -755,21 +815,26 @@ def _download_file(
     stream: bool = True,
```

Comment:
- `stream` is unused now
- progress tracking need to be implemented using the operation progress field, will be a breaking change.